react-accordion/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+// Type for accordion items
+type AccordionItem = {
+  id: number;
+  title: string;
+  content: string;
+};
+
+interface AccordionProps {
+  items: AccordionItem[];
+}
+
+export function Accordion({ items }: AccordionProps) {
+  const [activeIndex, setActiveIndex] = useState(-1);
+
+  const handleToggle = (index: number) => {
+    setActiveIndex(activeIndex === index ? -1 : index); // Toggle between open/close
+  };
+
+  return (
+    <div className="accordion">
```

Comment:
The purpose of this exercise is to get experience working with multiple components. Please split out the TopicCard component. A TopicCard should display _one_ topic, either opened or closed.